packages/aws-cdk-lib/aws-cloudwatch/lib/dashboard.ts

```diff
@@ -133,7 +133,11 @@ export class Dashboard extends Resource {
     }
 
     if (props.start !== undefined && props.defaultInterval !== undefined) {
-      throw ('both properties defaultInterval and start cannot be set at once');
+      throw new Error('both properties defaultInterval and start cannot be set at once');
+    }
+
+    if (props.end !== undefined && props.start === undefined) {
+      throw new Error('You must also specify a start if you specify an end');
```

Comment:
I think that a better error message could be `If you specify a value for end, you must also specify a value for start.` (taken from the [docs](https://docs.aws.amazon.com/cdk/api/v2/docs/aws-cdk-lib.aws_cloudwatch.Dashboard.html#:~:text=If%20you%20specify%20a%20value%20for%20end%2C%20you%20must%20also%20specify%20a%20value%20for%20start.))

Reply:
@lpizzinidev 

Thanks for your review. I changed!

https://github.com/aws/aws-cdk/pull/27124/commits/d3d0b2bb6413073e76a10837eecadbbc633680cf